app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/attributeStrategies.ts

```diff
@@ -0,0 +1,82 @@
+// In an OOP Language -
+// TypeScript
+interface FeatureAttributeFunctions {
+    isNumeric() : boolean
+    isString(): boolean
+}
+
+enum Attributes {
+    LABEL = "label",
+    TYPE = "type",
+    URI = "uri",
+    FREQUENCY = "frequency",
+    TIMESTAMP = "timestamp",
+    LOCALITY = "locality",
+    POSTALCODE = "postalcode",
+    REGIONS =  "regions",
+    TRANSMISSIONPOWER = "transmissionPower",
+    GEOMETRY = "geometry",
+}
+
+export class FeatureAttribute implements FeatureAttributeFunctions{
```

Comment:
This class should be abstract, or better, be an interface so you can potentially put it in the Redux state.